course/templates/course/course_base.html

```diff
@@ -75,6 +75,36 @@ <h4>{% trans "Calendar" %}</h4>
                 {% endif %}
             </div>
         {% endif %}
+        {% if query_language %}
+            {% language user_language %}
+            <div lang={{user_language}}>
+                <div class="alert alert-default">
+                    {% trans "You are browsing this course in different language than set in your preferences." %}
```

Comment:
This word choice is complicated as "preferences" here might refer to two different places. After this PR https://github.com/apluslms/a-plus/pull/632, it's possible to set multiple different language preferences for one user, one for the profile (affects the profile page and A+ front page etc) and one for each course separately (set from the language change symbol in course menu). 

So the word choice could be different depending on where the language is changed. When changed from profile, "set in your profile" is good, but what is a good way to indicate to user that the language is the one they have set for this specific course? 
If we would use one message, it could have some other word than "preferences" but I wouldn't use "profile" as it doesn't always come from profile.

Reply:
I understand. I think this is the right way to do it for now, I'd confused the places.

It's a little awkward though, as I'm afraid that if we have almost any word in a sentence like that (but especially "preferences"), a user will be looking for a "preferences" (or whatever) link to check the setting.

Reply:
That is true that it might be confusing for the user, I'll have to think a bit for better word choices.